internal/api/schema.go

```diff
@@ -167,3 +169,65 @@ func (a *API) DeleteVersion(ctx context.Context, in *stencilv1beta1.DeleteVersio
 		Message: message,
 	}, err
 }
+
+func (a *API) DetectSchemaChange(writer http.ResponseWriter, request *http.Request, pathParams map[string]string) error {
+	namespaceID := pathParams["namespaceId"]
+	schemaID := pathParams["schemaId"]
+
+	fromVersion := request.URL.Query().Get("from")
+	toVersion := request.URL.Query().Get("to")
+
+	versionList, err := a.schema.ListVersions(context.Background(), namespaceID, schemaID)
+	if err != nil || len(versionList) == 0 {
+		return fmt.Errorf("error getting version list - %s", err)
+	}
```

Comment:
```suggestion
	if err != nil {
		return fmt.Errorf("error getting version list - %s", err)
	}
	if len(versionList) == 0 {
	      return fmt.Errorf("got empty version list")
	}
```

Reply:
If we have used %s, then pass param err.Error()